bin/bootstrap.pl

```diff
@@ -47,26 +46,26 @@
     say "Please enter the github password for $GITHUB_USER";
     git( 'clone', $github_user_url . '/dw-free.git', $LJHOME );
 
-    configure_dw_upstream( 'dw-free' );
+    configure_dw_upstream('dw-free');
 }
 
 # now get dw-nonfree if it's not there *and* the user has asked for it
 if ( -d "$LJHOME/ext/dw-nonfree/.git" ) {
     say "Looks like you already have dw-nonfree checked out; skipping.";
 }
-elsif ( $DW_NONFREE ) {
+elsif ($DW_NONFREE) {
```

Comment:
Wait, why is it removing spaces in parens?

Reply:
I noticed it seemed to be doing that for method calls but not basic function calls. Maybe it's also omitting them for small runs of parenthesized text? Seems inconsistent.

Reply:
I think maybe it cares whether the thing in parens is a single thing. Class::Subclass is multiple things with an operator in between, but @$arrayref is one thing, "string that looks ( more, complicated" is one thing, etc.